### ISSUE TITLE
extractor.init errer

### ISSUE THREAD

**Issue by jodjo86** [GenomiqueENS/toulligQC#21]
I got this error with the last version of toulligqc (2.5)
```
folder=ONV-15-2023_Detection
toulligqc --report-name $folder --barcoding \
         --telemetry-source $folder/fastq/sequencing_telemetry.js \
         --sequencing-summary-source $folder/fastq/sequencing_summary.txt \
         --html-report-path $folder/QC_report.html \
         --barcodes barcode01:barcode24
ONV-15-2023_Detection/QC_report.html
ToulligQC version 2.5
* Initialize extractors
* Start Toulligqc info extractor
* End of Toulligqc info extractor (done in 0m0.00s)
* Start Sequencing telemetry extractor
* End of Sequencing telemetry extractor (done in 0m0.00s)
* Start Basecaller sequencing summary extractor
Traceback (most recent call last):
  File "/home/joel/miniforge3/envs/nano/bin/toulligqc", line 10, in <module>
    sys.exit(main())
  File "/home/joel/miniforge3/envs/nano/lib/python3.10/site-packages/toulligqc/toulligqc.py", line 388, in main
    extractor.init()
  File "/home/joel/miniforge3/envs/nano/lib/python3.10/site-packages/toulligqc/sequencing_summary_extractor.py", line 117, in init
    self.dataframe_1d['barcode_arrangement'].cat.add_categories([0, 'other barcodes', 'passes_filtering'],
  File "/home/joel/miniforge3/envs/nano/lib/python3.10/site-packages/pandas/core/accessor.py", line 112, in f
    return self._delegate_method(name, *args, **kwargs)
  File "/home/joel/miniforge3/envs/nano/lib/python3.10/site-packages/pandas/core/arrays/categorical.py", line 2893, in _delegate_method
    res = method(*args, **kwargs)
TypeError: Categorical.add_categories() got an unexpected keyword argument 'inplace'
```
I will soon attach a link with the telemetry files (they are a bit big).

**Comment by jodjo86** [comment]
https://api.onedrive.com/v1.0/shares/u!aHR0cHM6Ly8xZHJ2Lm1zL3UvcyFBdWNHeUZoaVpRdENoYTlPQnBkdjBDTGtPRGRrdkE_ZT1mY3pLRmE/root/content

**Comment by alihamraoui** [comment]
Hi @jodjo86,

Thank you for reporting this issue. It seems that the 'inplace' parameter and the append() function have been deprecated and removed in Pandas version 2.0.

I have addressed this in the commit 2c276c61994eee2cac69686e197184e99e67574e .

Should you encounter any further issues with the latest version of Pandas, I recommend reverting to Pandas version 1.5.3, which can be installed using the following command: `pip3 install pandas==1.5.3`.

Best regards,
Ali

**Comment by jodjo86** [comment]
thank you
